pom.xml

```diff
@@ -1750,10 +1749,16 @@
                <artifactId>jacoco-maven-plugin</artifactId>
                <version>${version.org.jacoco.plugin}</version>
             </plugin>
+            <plugin>
+               <!-- This is needed until https://issues.apache.org/jira/browse/MSHADE-462 is resolved. Once that is
+                    done then we can inherit the version from the Apache parent POM. -->
+               <groupId>org.apache.maven.plugins</groupId>
+               <artifactId>maven-shade-plugin</artifactId>
+               <version>3.4.1</version>
+            </plugin>
```

Comment:
You can just set (i.e override) the property they used in the parent dependencyManagement to the desired version, rather than add the full config and hard-coded version.

Reply:
Good idea!